lighty-applications/lighty-rnc-app-aggregator/README.md

```diff
@@ -95,16 +95,81 @@ To build and start the RNC lighty.io application using docker in the local envir
+   , then your path to this file in configuration.json becomes just "./initData.xml": 
+   ` "initialConfigData": {
+          "pathToInitDataFile": "./initData.xml",
+          "format": "xml"
+         }`
    
    Example configuration files are located on following path:  
-   `lighty-rnc-app/src/main/resources/*`
+   `lighty-rnc-app-docker/example-config/*`
 
 4. If the application was started successfully, then a log similar should be present in the console:  
    `INFO [main] (Main.java:81) - RNC lighty.io application started in 5989.108ms`
 
 5. Test the RNC lighty.io application. Default RESTCONF port is `8888`
 
+
+## Deployment via helm chart
+### Prerequisites
+* kubernetes cluster 1.15.11 (minikube / microk8s /..)
+* helm 2.17
+### Deploy
+To easily deploy Lighty RNC application to kubernetes we provide custom helm chart located in /lighty-rnc-app-helm/helm/.
+To install, make sure that the docker image defined in `values.yaml` is accessible, then run command:
+`helm install microk8s helm install --name lighty-rnc-app ./lighty-rnc-app-helm/`
+in `/lighty-rnc-app-helm/helm/` directory.
+### Providing startup configuration
+By default, the deployed application is started with custom configuration.json 
+(for more information check [lighty-controller](https://github.com/PANTHEONtech/lighty/tree/13.1.x/lighty-core/lighty-controller)).
+We supply this configuration file by passing kubernetes configmap (`configmaps.yaml`), which you can modify to your needs.
+To use the functionality of loading configuration data on startup, add new entry to configmaps.yaml:
+`initData: |
+     your initial yang modeled json/xml data
+`
+Then add:
+` "initialConfigData": {
+       "pathToInitDataFile": "{{ .Values.lighty.configDirectoryName }}/initData",
+       "format": "xml"/"json" depending on format
+      }` 
+entry to controller json node in lighty-config.json in `configmaps.yaml`.
+If everything was set up corectly, then your data will be loaded to controller on startup and appropriate listeners should be triggered.
+For example, if your initial json data contains node in netconf topology: 
+`{
+  "network-topology:network-topology": {
+    "topology": [
+      {
+        "topology-id": "topology-netconf",
+        "node": [
+          {
+            "node-id": "device1",
+            "netconf-node-topology:schemaless": false,
+            "netconf-node-topology:port": <port-of-device>,
+            "netconf-node-topology:tcp-only": false,
+            "netconf-node-topology:username": "admin",
+            "netconf-node-topology:password": "admin",
+            "netconf-node-topology:host": "<ip-of-device>",
+            "netconf-node-topology:keepalive-delay": 0
+          }
+        ]
+      }
+    ]
+  }
+}`
+and the device is running, the connection should be established upon startup.
+For testing purposes, you can use [lighty-netconf-simulator](https://github.com/PANTHEONtech/lighty-netconf-simulator)
+as a netconf device.
```

Comment:
Add a note that invalid configuration of init data leads to shutting down the lighty controller (or in another words not starting it at all...)